nautobot/extras/tests/test_filters.py

```diff
@@ -929,15 +929,36 @@ class JobQueueFilterSetTestCase(FilterTestCases.FilterTestCase, FilterTestCases.
     tenancy_related_name = "job_queues"
     generic_filter_tests = [
         ["name"],
+        ["context"],
+        ["secrets_group", "secrets_group__id"],
+        ["secrets_group", "secrets_group__name"],
     ]
 
     @classmethod
     def setUpTestData(cls):
         # create some job queues that do not have jobs attached to them
         # for has_jobs boolean filter
-        JobQueue.objects.create(name="Empty Job Queue 1", queue_type=JobQueueTypeChoices.TYPE_CELERY)
+        secrets_groups = (
+            SecretsGroup.objects.create(name="Secrets Group 1"),
+            SecretsGroup.objects.create(name="Secrets Group 2"),
+            SecretsGroup.objects.create(name="Secrets Group 3"),
+        )
+        JobQueue.objects.create(
+            name="Empty Job Queue 1", queue_type=JobQueueTypeChoices.TYPE_CELERY, secrets_group=secrets_groups[0]
```

Comment:
Should we disallow setting a secrets_group (or a context) on a TYPE_CELERY JobQueue?